PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -187,6 +148,7 @@ def miniAOD_customizeMC(process):
     process.patJetPartonMatch.matched = "prunedGenParticles"
     process.patJetPartonMatch.mcStatus = [ 3, 23 ]
     process.patJetGenJetMatch.matched = "slimmedGenJets"
+    process.patJetGenJetMatchAK8.matched =  "slimmedGenJetsAK8"
```

Comment:
This line is not needed since the `addJetCollection(...)` call for AK8 jets already has `genJetCollection = cms.InputTag('slimmedGenJetsAK8')`.